test/drivers/test_fluke_8846A.py

```diff
@@ -316,6 +316,192 @@ def test_measurement_diode(funcgen, dmm, rm):
     assert meas == pytest.approx(TEST_DIODE, abs=TEST_DIODE_TOL)
 
 
+@pytest.mark.parametrize(
+    "mode",
+    [
+        ("voltage_dc"),
+        ("current_dc"),
+        ("resistance"),
```

Comment:
Looking at the datasheet for the fluke, it does not support changing nplc on diode measurements (while the keithley does). I could add it to the fluke tests with the xfail mark if we want that coverage.

Reply:
What about removing it from the keithley, from the point of view of keeping the two driver as compatible as possible (until we having compelling reason to do otherwise)

Reply:
I've removed diode as a valid mode for which NPLC can be changed from the keithley driver.
I've also updated the tests accordingly
this is done as of 5d96f54